android/quest/build.gradle

```diff
@@ -138,7 +138,8 @@ android {
             applicationIdSuffix ".ecbis"
             versionNameSuffix "-ecbis"
             versionCode 4
-            versionName "0.1.5-rc1"
+            versionName "0.1.6-rc1"
+            buildConfigField "String[]", "SYNC_STRATEGIES", '{"Location", "Organization"}'
```

Comment:
Can we move this to the ApplicationConfig?
For reference https://github.com/opensrp/fhircore/issues/1778